cipher/rot13/rot13_test.go

```diff
@@ -64,7 +64,19 @@ func TestRot13Decrypt(t *testing.T) {
 func assertRot13Output(t *testing.T, input, expected string) {
 	actual := rot13(input)
 	if actual != expected {
-		t.Fatalf("With input string '%s' was expecting '%s' but actual was '%s'",
+		t.Fatalf("With input string %q was expecting %q but actual was %q",
```

Comment:
Nice! I like the quote formatter - I recently learnt about them.

Reply:
Eheh, yes I saw it and I said it was a good opportunity to change it with %q. It's actually a great formatting verb for strings in my opinion